pandas/io/json/normalize.py

```diff
@@ -241,10 +272,12 @@ def _recursive_extract(data, path, seen_meta, level=0):
         else:
             for obj in data:
                 recs = _pull_field(obj, path[0])
+                recs = [nested_to_record(r, sep=sep,
+                                         max_level=max_level)
+                        if isinstance(r, dict) else r for r in recs]
```

Comment:
this is odd to do, no? nested_to_record handles dicts yes?

Reply:
If it is a dict we will flatten it using `nested_to_record` if not use the value as is.

Reply:
Can you show an example where this is hit? Are we sure we aren't changing behavior for max_level=None?

Reply:
`max_level` none is `json_normlize` default behavior. Also how would the user say "flatten at all level"